<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_js">
     <property name="title">
      <string>JS</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_run_js">
          <property name="text">
           <string>run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cb_auto_js">
          <property name="text">
           <string>auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_load_js">
          <property name="text">
           <string>load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_save_js">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_default_js">
          <property name="text">
           <string>default</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MultiEdit" name="me_js" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="MyBrowser" name="browser_widget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiEdit</class>
   <extends>QWidget</extends>
   <header>multiedit.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyBrowser</class>
   <extends>QWidget</extends>
   <header>mybrowser.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
